src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/models/ReportDraft.kt

```diff
@@ -0,0 +1,11 @@
+package org.vaccineimpact.orderlyweb.models
+
+import java.time.Instant
+
+data class ReportDraft
```

Comment:
There is getting to be quite a proliferation of models here. I guess this is because you just want to return the bits you need for drafts from the repo? Alternatives would be to return ReportVersions, just with empty customFields and tags, or tuples of BasicReportVersion. That might be less clear though.

Reply:
Yeah so this is the bit I am not very happy with but also can't find a better solution. 

I def don't think we should re-use models where we just don't populate parts of it - I've worked with a codebase which took that approach before and it makes things very confusing down the line! And there's also a string property - `latestVersion` - which isn't needed for the `ReportDraft`.

Not sure what you mean by tuples of `BasicReportVersion`? `BasicReportVersion` both has a property we don't need - `latestVersion` - and is missing one we do need - `changelog`, which we do need. So my dilemma was do I write the most efficient sql query to get back exactly the data we need but add yet another model for report versions, or do I re-use existing classes but do extra joins, make extra db calls, and do extra in memory data munging to combine changelogs and report drafts.

I'm still not too happy about the end result and wondering if there is some further re-organisation of report version models available. I suppose one thing we've shied away from is inheritance - perhaps in this instance it would make things nicer?